src/pages/resultsView/cancerSummary/CancerSummaryChart.tsx

```diff
@@ -245,65 +325,210 @@ export class CancerSummaryChart extends React.Component<CancerSummaryChartProps,
+            x: this.categoryAxisDomainPadding()
+        };
+    }
+
+    private barChartHeight() {
+        return 400;
+    }
 
-        // if we're not showing result broken down the alterations then we need to merge
+    @computed get scatterChartHeight() {
+        return this.scatterChartExtent;
+    }
+
+    @computed get scatterChartExtent() {
+        let miscPadding = 100; // specifying chart width in victory doesnt translate directly to the actual graph size
+        const profiledDataTypesLength = _.uniqBy(this.scatterData,datum=>datum.y).length
+        if (profiledDataTypesLength > 0) {
+            return profiledDataTypesLength * (this.barWidth() + this.barSeparation()) + miscPadding;
+        } else {
+            return miscPadding;
+        }
+    }
+
+    @computed get scatterPlotTopPadding() {
+        // subtract 100 to plot scatter p\lot close to bar plot
+        return this.barChartHeight() - 100;
+    }
+
+    @computed get scatterData() {
+        return _.map(this.props.alterationTypeDataCounts, datum => {
+            return {
+                ...datum,
+                symbol: datum.profiledCount > 0 ? "plus" : "minus"
+            }
+        });
+    }
+
+    @computed get barPlots() {
+         // if we're not showing result broken down the alterations then we need to merge
         // data across alterations (and hide legend down below)
-        const alterationData = (this.props.hideGenomicAlterations) ?
-            [mergeAlterationDataAcrossAlterationTypes(this.props.data)] : this.props.data
+        const alterationData =  (this.props.hideGenomicAlterations) ?
+        [mergeAlterationDataAcrossAlterationTypes(this.props.data)] : this.props.data
 
+        return alterationData.map((data: any, i: number) => {
+            return <VictoryBar style={{ data: { width: this.barWidth } }} events={this.barPlotEvents} data={data} key={i} />;
+        })
+    }
+
+    @autobind private getChart() {
         return (
-            <div data-test="cancerTypeSummaryChart">
-                <div style={this.overflowStyle} className="borderedChart">
-                    <div ref={(el:HTMLDivElement)=>this.scrollPane=el} style={{overflowX:'auto', overflowY:'hidden'}}>
-                    {
-                        (this.tooltipModel) && (this.buildTooltip(this.tooltipModel))
-                    }
-                    <VictoryChart width={this.width} height={this.height} theme={CBIOPORTAL_VICTORY_THEME}
-                                  padding={{bottom:this.bottomPadding, top:this.topPadding, left:this.leftPadding, right:this.rightPadding }} domainPadding={{x: 20, y: 20}}
-                                  containerComponent={<VictoryContainer containerRef={(ref: any) => this.svgContainer = ref} responsive={false}/>}
+            <div style={this.overflowStyle} className="borderedChart">
+                <div style={{ overflowX: 'auto', overflowY: 'hidden' }}>
+                    <svg
```

Comment:
what's reason for all this extra wrapping?

Reply:
this is to have both bar plot and scatter plot together